File: solutions/beecrowd/2591/2591.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

c = int(input())
for _ in range(c):
    line = input().strip()
    first_a = line.find('meka') - 1
    second_a = len(line) - first_a - 6
    output(f'k{"a" * first_a * second_a}\n')

File: solutions/beecrowd/2591/2591.lua
local c = tonumber(io.read("*n"))
io.read() -- go to next line

for i=1,c do
    local line = io.read()
    local first_a, _ = string.find(line, "meka")
    first_a = first_a - 2
    local second_a = string.len(line) - first_a - 6

    print(string.format('k%s', string.rep("a", first_a * second_a)))
end
